Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/ComboBoxInput.swift

```diff
@@ -79,29 +74,34 @@ struct ComboBoxInput: View {
     }
     
     var body: some View {
-        HStack {
-            Text(displayedValue)
-                .accessibilityIdentifier("\(element.label) Combo Box Value")
-                .frame(maxWidth: .infinity, alignment: .leading)
-                .foregroundStyle(!selectedValue.isNoValue ? .primary : .secondary)
-            if let _ = selectedValue.codedValue, !isRequired {
-                // Only show clear button if we have a value
-                // and we're not required. (i.e., Don't show clear if
-                // the field is required.)
-                XButton(.clear) {
-                    embeddedFeatureFormViewModel.focusedElement = element
-                    defer { embeddedFeatureFormViewModel.focusedElement = nil }
-                    updateValueAndEvaluateExpressions(nil)
+        Button {
+            embeddedFeatureFormViewModel.focusedElement = element
+            isPresented = true
+        } label: {
+            HStack {
+                Text(displayedValue)
+                    .accessibilityIdentifier("\(element.label) Combo Box Value")
+                Spacer()
+                if let _ = selectedValue.codedValue, !isRequired {
+                    // Only show clear button if we have a value
+                    // and we're not required. (i.e., Don't show clear if
+                    // the field is required.)
+                    XButton(.clear) {
+                        embeddedFeatureFormViewModel.focusedElement = element
+                        defer { embeddedFeatureFormViewModel.focusedElement = nil }
+                        updateValueAndEvaluateExpressions(nil)
+                    }
+                    .accessibilityIdentifier("\(element.label) Clear Button")
+                    .buttonStyle(.plain)
```

Comment:
This is already the button style of `XButton`, except on visionOS. Is this really needed?